.cbmc-batch/include/make_common_data_structures.h

```diff
@@ -47,3 +47,5 @@ void ensure_trace_has_allocated_records(struct aws_array_list *trace, size_t max
 /* Non-deterministically allocates a aws_cryptosdk_keyring structure */
 void ensure_cryptosdk_keyring_has_allocated_members(
     struct aws_cryptosdk_keyring *keyring, const struct aws_cryptosdk_keyring_vt *vtable);
+/* Non-deterministically allocates a aws_cryptosdk_keyring_vt structure with a valid name*/
+void ensure_nondet_allocate_vtable_members(struct aws_cryptosdk_keyring_vt *vtable, size_t max_len);
```

Comment:
This name seems like it would lead to confusion between keyring vtables and CMM vtables. For example in `.cbmc-batch/jobs/aws_cryptosdk_cmm_base_init/aws_cryptosdk_cmm_base_init_harness.c` line 31, you've passed a CMM vtable pointer where the function declaration suggests it should be a keyring vtable pointer. I assume this works out since `ensure_nondet_allocate_vtable_members` only accesses the `name` fields of the vtables, which happen to be identically positioned in the respective structs, but I'd prefer to avoid relying on that.

Reply:
Good point! I've updated and made two separate functions, one for cmm and one for keyring_trace. They do the same thing right now but who knows, maybe down the line they'll diverge. Just waiting for the checks to go through now.